hola.py

```diff
@@ -51,6 +51,18 @@ def sumar_lista_doble(lista):
 
     return suma
 
+def insertion_sort(lista):
+    for i in range(1, len(lista)):
+        k = lista[i]
```

Comment:
```suggestion
        key = lista[i]
```

Cambia la variable `k` por `key`